DiceKit/FrequencyDistribution.swift

```diff
@@ -222,9 +239,25 @@ extension FrequencyDistribution {
         }
     }
     
-    // TODO: public func divide(x: FrequencyDistribution) -> FrequencyDistribution
-    // Needed to support power(-n)
-    
+    public func divide(y: FrequencyDistribution) -> FrequencyDistribution {
+        guard let initialK = orderedOutcomes.first, lastK = orderedOutcomes.last else {
+            return .additiveIdentity
+        }
+        guard let firstY = y.orderedOutcomes.first, lastY = y.orderedOutcomes.last, firstYFrequency = y[firstY] where firstYFrequency != 0.0 else {
+            fatalError("Divide by zero! :(")
+        }
+        var xFrequencies: FrequenciesPerOutcome = [:]
+        var p: Frequency
+        for k:Outcome in initialK...(lastK + lastY) {
+            p = 0.0
```

Comment:
It is used outside of the loop? I don't see `p` referenced anywhere but inside of it.